DEVELOPING_A_RUNTIME.md

```diff
@@ -267,16 +267,18 @@ This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaSc
 ### `Lambda`
 
 ```ts
-import { Lambda } from '@now/build-utils'
+import { Lambda } from '@now/build-utils';
 ```
 
-This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), called a Serverless Function, that can be created by supplying `files`, `handler`, `runtime`, and `environment` as an object to the [`createLambda`](#createlambda) helper. The instances of this class should not be created directly. Instead use a call to [`createLambda`](#createlambda).
+This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), called a Serverless Function, that can be created by supplying `files`, `handler`, `runtime`, and `environment` as an object to the [`createLambda`](#createlambda) helper, the properties `memory` and `maxDuration` are optional. The instances of this class should not be created directly. Instead use a call to [`createLambda`](#createlambda).
```

Comment:
I would either make the optional part be a separate sentence, or inside parenthesis:

```suggestion
This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), called a Serverless Function, that can be created by supplying `files`, `handler`, `runtime`, and `environment` as an object to the [`createLambda`](#createlambda) helper (the properties `memory` and `maxDuration` are optional). The instances of this class should not be created directly. Instead use a call to [`createLambda`](#createlambda).
```